docs/trace-prefetch.md

```diff
@@ -8,15 +8,38 @@ There are many ways to do prefetch, for instance, we can simply read extra data
 
 Another way is to [prioritize files and use landmarks](https://github.com/containerd/stargz-snapshotter/blob/master/docs/stargz-estargz.md#prioritized-files-and-landmark-files), which is already adopted in Google's stargz. The storage engine runtime will prefetch the range where prioritized files are contained. And finally this information will be leveraged for increasing cache hit ratio and mitigating read overhead.
 
-In this article we are about to introduce a new prefetch mechanism based on time sequenced I/O patterns (trace). This mechanism has been integrated as a feature into `ctr record-trace` command.
+In this article we are about to introduce two prefetch modes in overlayBD. One is to set prioritize files, another is a new prefetch mechanism based on time sequenced I/O patterns (trace).
+These two mechanisms have been integrated as a feature into `ctr record-trace` command.
 
-## Trace Prefetch
+## Prefetch Mode
+
+### Prioritize Files
+
+Setting prioritize files is a simple way to improve container's cold start time. It is suitable for the condition where the target files needed be fully loaded.
+
+When overlaybd device has been created, it will parse the prioritize filelist and analyze the filesystem via libext4 before mounting, downloading the target files to overalybd's cache.
```

Comment:
When overlaybd device is created, it will parse the prioritize filelist and analyze the filesystem via libext4 before mounting, then download the target files to overalybd's cache.